src/web/client/dal/remoteSaveProvider.ts

```diff
@@ -189,9 +187,6 @@ async function saveDataToDataverse(
                 new Date().getTime() - requestSentAtTime,
                 fileExtensionType
             );
```

Comment:
We need to replace this with a success event right?

Reply:
I will add the event name in successTelemetry itself rather than creating another event.